### PR TITLE
Update love package: repo URL updated

### PR DESCRIPTION
**Pull request by paulopacitti**
## Introduction
- This is my first contribution to a open-source project, and I'm very happy in contributing with **brew** :sparkles: :beer: So, it's possible that I made some mistake when requesting this pull request 😬  Feel free to comment below what I forgot to do or what I didn't do in the right way;

## Description:
- Recently, [Löve](https://love2d.org/) 💟   migrated from BitBucket to GitHub, so `brew cask install love` was not working anymore;
- I changed the URL to the current repo;
- `brew cask style --fix {{cask_file}}` report offense because the new URL didn't match with the last one. 
```ruby
paulopacitti お茶 homebrew-cask ⚡️ brew cask style --fix love
== /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/Casks/love.rb ==
C:  5:  3: bitbucket.org/rude/love/ does not match github.com/love2d/love/releases/download/

1 file inspected, 1 offense detected
Error: Style check failed.
```
I didn't know how to fix it, some ideas on it?

## Checklist
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by paulopacitti** [comment]
I managed to fix `brew cask style --fix {{cask_file}}` fixing the ` ... was verified as official when first introduced to the cask` comment. I dunno if this is this is the right way of fixing it 😅

File: Casks/love.rb [hunk 1]
```diff
@@ -2,8 +2,7 @@
   version "11.3"
   sha256 "f506af5a6443a559d8edf9b165fd817acb5accdf9873a3be9baf41ab09e0d2d9"
 
-  # bitbucket.org/rude/love/ was verified as official when first introduced to the cask
-  url "https://bitbucket.org/rude/love/downloads/love-#{version}-macos.zip"
+  url "github.com/love2d/love/releases/download/#{version}/love-#{version}-macos.zip"
   appcast "https://love2d.org/"
```
Comment (miccal) [inline Casks/love.rb:6]: Please change these lines to:
```
# github.com/love2d/love/ was verified as official when first introduced to the cask
url "https://github.com/love2d/love/releases/download/#{version}/love-#{version}-macos.zip"
appcast "https://github.com/love2d/love/releases.atom"
```

Comment (paulopacitti) [reply Casks/love.rb]: Done!

**Comment by vitorgalvao** [comment]
Welcome, and congrats on your first contribution.

Thank you for the fix.

**Pull request merged**